src/Api/Catalog/CatalogDomain.php

```diff
@@ -0,0 +1,74 @@
+<?php
+namespace ShoppingFeed\Sdk\Api\Catalog;
+
+use ShoppingFeed\Sdk\Resource\AbstractDomainResource;
+
+class CatalogDomain extends AbstractDomainResource
+{
+    /**
+     * @var string
+     */
+    protected $resourceClass = CatalogResource::class;
+
+    /**
+     * @var null|CatalogResource
+     */
+    protected $catalog = null;
+
+    public function askForFeedImport($force = null)
```

Comment:
Si tu peux ajouter les type hint de retour en docblock. On ne type pas pour rester compatible php5.6 mais c'est quand même bien de figer/documenter le typage.

Reply:
```suggestion
    public function requestFeedImport($force = null)
```
Serait plus clair. et il faut documenter le $force pour savoir a quoi il sert (dans le code et dans la doc du repo)

Reply:
Pas sur que le retour sur cette méthode soit vraiment utile, si c'est un process asynchrone qui est demandé.

Reply:
ça peut être utile, si jamais on veut avoir la réponse : un 202, si tout se passe bien.

Reply:
Il me semble que le comportement de base pour les call asynchrone d'opération (ex: [les opérations orders](https://github.com/shoppingflux/php-sdk/blob/master/src/Api/Order/OrderOperation.php)) :
- si tout est ok > on retourne rien
- sinon > Exception